src/cts/src/TechChar.cpp

```diff
@@ -1332,18 +1327,20 @@ void TechChar::updateBufferTopologies(TechChar::SolutionData& solution)
 
 std::vector<size_t> TechChar::getCurrConfig(const SolutionData& solution)
 {
+  std::stringstream tmp;
```

Comment:
Move inside the if-debugCheck bloc - there is no need to construct it when it isn't used.  In general variables should be in the smallest possible scope and nearest to their first point of use.  Similarly in functions below.